Nightfoxy new ns loop variables

Nightfoxy opened the pull request:
Add instructions for new NS site builders to add loop-specific variables by pointing them to the Retrofit Existing NS page.

Nightfoxy commented:
bump

ps2 reviewed (changes requested):
The url referenced here (http://nightscout.github.io/nightscout/update_user/) no longer works. Feel free to update it and re-request a review.

Nightfoxy commented:
fixed. The URL was supposed to point people to the retrofit instructions in loop docs. Somehow the NS github domain was put in. I fixed it. Basically, people need to know to return to loop docs for loop-specific config instructions since NS docs are not including it.

ELUTE commented:
@ps2 is this one good to go now?

marionbarker commented:
Kenny - I have captured your requested change and will insert it into the markdown cleaned up version.
